Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
-    'common',
+    'common/common_traits',
```

Comment:
I didn't say separate crates but modules, like `common/traits.rs` and `common/arith_utils.rs`